src/config.rs

```diff
@@ -214,13 +216,20 @@ impl ConfPred {
         } else {
             vec![PathBuf::from("/var/tmp")]
         };
+        let mtimedbfile = sel!(cli,
+                               toml,
+                               predict,
+                               mtimedbfile,
+                               (),
+                               String::from("/var/cache/edb/mtimedb"))?;
```

Comment:
I'd say there's not much readability win in assigning this here ? I'd say move it inline inside the ` Ok` like for stats. Whatever the choice, should be the same for stats and pred.